common/cloudant_base_test.go

```diff
@@ -232,4 +235,72 @@ var _ = Describe(`Cloudant custom base service UT`, func() {
 		Expect(cloudant.UserAgent).To(ContainSubstring(runtime.GOOS))
 		Expect(cloudant.UserAgent).To(ContainSubstring(runtime.GOARCH))
 	})
+
+	It("Validates cookie jar enabled for all auths", func() {
+		cloudant, err := NewBaseService(&core.ServiceOptions{
```

Comment:
Let's do it in a loop. Build a slice of `[]core.Authenticator` with all the auth in there and then just `range` over it. Normally in golang this is done in table driven tests, but ginkgo added those only in v2

Reply:
Added loop in [f87eb79](https://github.com/IBM/cloudant-go-sdk/pull/398/commits/f87eb79a1ee5173f386fd8b201df860047335456)